htmlReport/css/coverage.css

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Remove this and all other files in `htmlReport` and add it to .gitignore.
Probably best to remove this commit. https://github.com/gammaStrahlung/monopoly_server/pull/41/commits/265accf6a9f31e2a582969367f80c19ca56d44ad